src/python/WMCore/WMSpec/Steps/Fetchers/PileupFetcher.py

```diff
@@ -70,15 +74,21 @@ def _queryDbsAndGetPileupConfig(self, stepHelper, dbsReader, fakeSites):
             # each dataset input can generally be a list, iterate over dataset names
             blockDict = {}
```

Comment:
Good question, maybe @ericvaandering  can shed some lights on here. I believe there could be a case like "stepHelper.mc.pileup", no? I'm probably wrong, since it works for any type (mc, data) of pileup...